src/assets/stylesheets/modern/main/extensions/mkdocstrings/_python.scss

```diff
@@ -0,0 +1,275 @@
+  --doc-symbol-class-fg-color: #0550ae;
+  --doc-symbol-type_alias-fg-color: #0550ae;
+  --doc-symbol-module-fg-color: #5cad0f;
+
+  --doc-symbol-parameter-bg-color: #829bd11a;
+  --doc-symbol-type_parameter-bg-color: #829bd11a;
+  --doc-symbol-attribute-bg-color: #9538001a;
+  --doc-symbol-function-bg-color: #8250df1a;
+  --doc-symbol-method-bg-color: #8250df1a;
+  --doc-symbol-class-bg-color: #0550ae1a;
+  --doc-symbol-type_alias-bg-color: #0550ae1a;
+  --doc-symbol-module-bg-color: #5cad0f1a;
+}
+
+[data-md-color-scheme="slate"] {
+  --doc-symbol-parameter-fg-color: #829bd1;
+  --doc-symbol-type_parameter-fg-color: #829bd1;
+  --doc-symbol-attribute-fg-color: #ffa657;
+  --doc-symbol-function-fg-color: #d2a8ff;
+  --doc-symbol-method-fg-color: #d2a8ff;
+  --doc-symbol-class-fg-color: #79c0ff;
+  --doc-symbol-type_alias-fg-color: #79c0ff;
+  --doc-symbol-module-fg-color: #baff79;
+
+  --doc-symbol-parameter-bg-color: #829bd11a;
+  --doc-symbol-type_parameter-bg-color: #829bd11a;
+  --doc-symbol-attribute-bg-color: #ffa6571a;
+  --doc-symbol-function-bg-color: #d2a8ff1a;
+  --doc-symbol-method-bg-color: #d2a8ff1a;
+  --doc-symbol-class-bg-color: #79c0ff1a;
+  --doc-symbol-type_alias-bg-color: #79c0ff1a;
+  --doc-symbol-module-bg-color: #baff791a;
+}
+
+code.doc-symbol {
+  border-radius: .1rem;
+  font-size: .85em;
+  padding: 0 .3em;
+  font-weight: bold;
+}
+
+code.doc-symbol-parameter,
+a code.doc-symbol-parameter {
+  color: var(--doc-symbol-parameter-fg-color);
+  background-color: var(--doc-symbol-parameter-bg-color);
+}
+
+.md-content code.doc-symbol-parameter::after {
+  content: "param";
+}
+
+.md-sidebar code.doc-symbol-parameter::after {
+  content: "P";
+}
+
+code.doc-symbol-type_parameter,
+a code.doc-symbol-type_parameter {
+  color: var(--doc-symbol-type_parameter-fg-color);
+  background-color: var(--doc-symbol-type_parameter-bg-color);
+}
+
+.md-content code.doc-symbol-type_parameter::after {
+  content: "type-param";
+}
+
+.md-sidebar code.doc-symbol-type_parameter::after {
+  content: "T";
+}
+
+code.doc-symbol-attribute,
+a code.doc-symbol-attribute {
+  color: var(--doc-symbol-attribute-fg-color);
+  background-color: var(--doc-symbol-attribute-bg-color);
+}
+
+.md-content code.doc-symbol-attribute::after {
+  content: "attr";
+}
+
+.md-sidebar code.doc-symbol-attribute::after {
+  content: "A";
+}
+
+code.doc-symbol-function,
+a code.doc-symbol-function {
+  color: var(--doc-symbol-function-fg-color);
+  background-color: var(--doc-symbol-function-bg-color);
+}
+
+.md-content code.doc-symbol-function::after {
+  content: "func";
+}
+
+.md-sidebar code.doc-symbol-function::after {
+  content: "F";
+}
+
+code.doc-symbol-method,
+a code.doc-symbol-method {
+  color: var(--doc-symbol-method-fg-color);
+  background-color: var(--doc-symbol-method-bg-color);
+}
+
+.md-content code.doc-symbol-method::after {
+  content: "meth";
+}
+
+.md-sidebar code.doc-symbol-method::after {
+  content: "M";
```

Comment:
note: Note that I differentiate between symbol types appearing in front of headings in the main area and in the table of contents (shorter, single letter for ToC). We might want to use proper names for the main area, like `method` instead of `meth`, `attribute` instead of `attr`, `function` instead of `func`, `module` instead of `mod`, etc.